tests/test_cmd_animation.sh

```diff
@@ -82,6 +82,17 @@ pushd ${TMP_DIR}
   out=$("${AVIFENC}" -s 8 "${INPUT_Y4M_0}" "${INPUT_Y4M_1}" -o "${ENCODED_FILE}")
   findstr "Encoding frame 0 [1/25 ts] color quality [60 (Medium)], alpha quality [60 (Medium)]" "${out}"
   findstr "Encoding frame 1 [1/25 ts] color quality [60 (Medium)], alpha quality [60 (Medium)]" "${out}"
+
+  # Test updating of tiling options.
```

Comment:
To test the updating of tiling options, I have to change avifenc.c to report the tiling options for each frame.